Makefile

```diff
@@ -0,0 +1,34 @@
+ORG = garethjevans
+IMAGE = $(ORG)/test-jaxrs-tomee
```

Comment:
Your user name is in here still.

We have a goal for adding Makefiles to all the project. I don't personally use them, but again I think this is probably a good topic for discussion. What would people want/expect in a Makefile. I'll start a thread on Slack.

In the meantime, maybe we just hold off on adding the Makefile.

Reply:
removed the makefile